services/app-web/src/components/SubmissionSummarySection/UploadedDocumentsTable/UploadedDocumentsTable.tsx

```diff
@@ -166,6 +183,18 @@ export const UploadedDocumentsTable = ({
                                     : ''}
                             </td>
                             <td>{documentCategory}</td>
+                            {showSharedInfo
+                                ? packagesWithSharedRateCerts &&
+                                  packagesWithSharedRateCerts.map((item) => (
+                                      <td>
+                                          <NavLink
+                                              to={`/submissions/${item.packageId}`}
+                                          >
+                                              {item.packageName}
+                                          </NavLink>
```

Comment:
Users can link to rate certifications in draft submissions where the package name could change and submitted packages could be unlocked and changes could also affect the package the name.

The upside is not having to make the query for the linked submissions to get the name. The downside is users may get confused if the linked package names don't match what is on the dashboard and when visiting the submission. I think in this case it might be good to just do the query and get that data to get the most up-to-date package name for the linked submission. @haworku @macrael any comments on this? Maybe I'm overthinking it? 

Also if the user had visited the dashboard page. I think the queried data would be cached anyways and not make another api call for the data. That would cut down on the actual api calls.

Reply:
Hey, @JasonLin0991 I think you are talking about the ratename - this is about the package name (looks something like MCR-MN-PMAP). 

@rswerve and I looked at this together last week at this together last week and thought there were few options 1. adjust what we store in the proto 2. make another request on the review-submit page to the index api -  display this link and handle error states if things go wrong 3.  adjust the initial  fetch draft submission graphql query to also return all related packages. 

We chose #1 because this was intended to be a quick feature, we know these types of associations between postgres and proto are already a hack, and we didn't see any major drawbacks.

Reply:
Ah, I see what you are saying, thanks for clarifying.

Reply:
Addressed in [6376b81](https://github.com/CMSgov/managed-care-review/pull/1349/commits/6376b81379132a04b0f8a41b76e7971cf3fc4549)